DependencyInjection/MonologExtension.php

```diff
@@ -89,7 +89,8 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('monolog.handlers_to_channels', $handlersToChannels);
 
             if (PHP_VERSION_ID < 70000) {
-                $this->addClassesToCompile(array(
+                $this->addClassesToCompile(
+                    [
```

Comment:
The previous version was better `$this->addClassesToCompile([`,
please keep the same indentation, and the same "style".

Reply:
Okays - will fix :+1: